### PR TITLE
DX-1721: Re-do MAC payerReference

### PR DESCRIPTION
**Pull request by arebra**
Removed the code example and hyperlinked them back to the payment order requests instead.

### CODE REVIEW

File: _includes/enterprise-payer-reference.md [hunk 1]
```diff
@@ -1,55 +1,40 @@
...
-|                  | └─➔&nbsp;`email`                   | `string`     | The e-mail address of the payer. Increases the chance for [frictionless 3-D Secure 2 flow][3d-secure-2].                                         |
-|                  | └─➔&nbsp;`msisdn`                  | `string`     | The mobile phone number of the payer. The mobile number must have a country code prefix and be 8 to 15 digits in length. The field is related to [3d-secure-2][3d-secure-2].            |
-|                  | └─➔&nbsp;`payerReference`                     | `string`     | A reference used in Enterprise and Payments integrations to recognize the payer in the absence of SSN and/or a secure login.                                                                                                                                                                                                                               |
-
-[3d-secure-2]: /checkout-v3/enterprise/features/core/3d-secure-2
-[enterprise-enter-ssn]: /assets/img/checkout/enterprise-enter-ssn.png
+[3d-secure-2]: /checkout-v3/mac/features/core/3d-secure-2
+[entreprise-enter-ssn]: /assets/img/checkout/mac-enter-ssn.png
```
Comment (trieutran91) [inline _includes/enterprise-payer-reference.md:36]:
```suggestion
[enterprise-enter-ssn]: /assets/img/checkout/enterprice-enter-ssn.png
```

This should get it past the verify check

File: _includes/enterprise-payer-reference.md [hunk 2]
```diff
@@ -1,55 +1,40 @@
...
-|                  | └─➔&nbsp;`email`                   | `string`     | The e-mail address of the payer. Increases the chance for [frictionless 3-D Secure 2 flow][3d-secure-2].                                         |
-|                  | └─➔&nbsp;`msisdn`                  | `string`     | The mobile phone number of the payer. The mobile number must have a country code prefix and be 8 to 15 digits in length. The field is related to [3d-secure-2][3d-secure-2].            |
-|                  | └─➔&nbsp;`payerReference`                     | `string`     | A reference used in Enterprise and Payments integrations to recognize the payer in the absence of SSN and/or a secure login.                                                                                                                                                                                                                               |
-
-[3d-secure-2]: /checkout-v3/enterprise/features/core/3d-secure-2
-[enterprise-enter-ssn]: /assets/img/checkout/enterprise-enter-ssn.png
+[3d-secure-2]: /checkout-v3/mac/features/core/3d-secure-2
```
Comment (trieutran91) [inline _includes/enterprise-payer-reference.md:35]:
```suggestion
```

Not in use

**Review by trieutran91 — CHANGES_REQUESTED** [review]
Just the renaming that causes trouble

File: _includes/enterprise-payer-reference.md [hunk 3]
```diff
@@ -1,55 +1,39 @@
...
-|                  | └─➔&nbsp;`email`                   | `string`     | The e-mail address of the payer. Increases the chance for [frictionless 3-D Secure 2 flow][3d-secure-2].                                         |
-|                  | └─➔&nbsp;`msisdn`                  | `string`     | The mobile phone number of the payer. The mobile number must have a country code prefix and be 8 to 15 digits in length. The field is related to [3d-secure-2][3d-secure-2].            |
-|                  | └─➔&nbsp;`payerReference`                     | `string`     | A reference used in Enterprise and Payments integrations to recognize the payer in the absence of SSN and/or a secure login.                                                                                                                                                                                                                               |
-
-[3d-secure-2]: /checkout-v3/enterprise/features/core/3d-secure-2
-[enterprise-enter-ssn]: /assets/img/checkout/enterprise-enter-ssn.png
+[enterprise-enter-ssn]: /assets/img/checkout/enterprice-enter-ssn.png
```
Comment (trieutran91) [inline _includes/enterprise-payer-reference.md:35]:
```suggestion
[enterprise-enter-ssn]: /assets/img/checkout/enterprise-enter-ssn.png
```

Haha ain't winning no price for enterpriCe

**Review by trieutran91 — APPROVED** [review]

File: _includes/enterprise-payer-reference.md (hunk 3 above)
Comment (trieutran91) [inline _includes/enterprise-payer-reference.md:35]:
```suggestion
[enterprise-enter-ssn]: /assets/img/checkout/enterprise-enter-ssn.png
```

Haha!  Ain't winning no price for enterpriCe!

**Review by trieutran91 — CHANGES_REQUESTED** [review]

File: _includes/enterprise-payer-reference.md (hunk 3 above)
Comment (arebra) [reply _includes/enterprise-payer-reference.md]: :D

**Review by trieutran91 — APPROVED** [review]

**Pull request merged**